packages/cashier/src/components/funds-protection/__tests__/funds-protection.spec.tsx

```diff
@@ -1,10 +1,13 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
+import { APIProvider } from '@deriv/api';
 import FundsProtection from '../funds-protection';
 
 describe('FundsProtection component tests', () => {
     it('should render the component', () => {
-        render(<FundsProtection />);
+        render(<FundsProtection />, {
+            wrapper: ({ children }) => <APIProvider>{children}</APIProvider>,
```

Comment:
i have a concern and a question about this
concern: changing WS to API can be confusing. so isn't it better to use WSProvider instead of APIProvider?
question: is this give WS access to this children?

Reply:
@ali-hosseini-deriv I'm using the `API` keyword to keep it consistent because the package name is `@deriv/api` and also our library name is `@deriv/deriv-api`, I've also renamed the `useWS` hook to the `useFetch` for the same reason.

> is this give WS access to this children?

No, It will wrap the children with `QueryClientProvider` to provide the caching system. However, in the future (very very long future 😁) when we have our socket connection here it will be providing the connection itself to the children as well.

Reply:
got it
thanks for clarification
still i think websocket keywords are more clear than this. but its just my opinion and i'm ok with this also

Reply:
@ali-hosseini-deriv Sure I will check on this with @mohsen-deriv and his roadmap for `@deriv/deriv-api` 🙇🏻